index.html

```diff
@@ -55,6 +55,12 @@ <h2>Privacy Protections Tests</h2>
 	<li><a href='http://privacy-test-pages.glitch.me/privacy-protections/surrogates/'>Surrogates</a></li>
 </ul>
 
+<h2>Autofill</h2>
+<ul>
+	<li><a href="./autofill/modal.html">Autofill form within modal</a></li>
```

Comment:
Let's make it clear that it's an *email* autofill.
```suggestion
	<li><a href="./autofill/modal.html">Email autofill form within modal</a></li>
```